src/webui/www/private/index.html

```diff
@@ -120,6 +120,7 @@ <h1 class="applicationTitle">qBittorrent Web User Interface <span class="version
         </li>
         <li class="separator"><a href="#DownloadLimit"><img src="theme/kt-set-max-download-speed" alt="QBT_TR(Limit download rate...)QBT_TR[CONTEXT=TransferListWidget]"/> QBT_TR(Limit download rate...)QBT_TR[CONTEXT=TransferListWidget]</a></li>
         <li><a href="#UploadLimit"><img src="theme/kt-set-max-upload-speed" alt="QBT_TR(Limit upload rate...)QBT_TR[CONTEXT=TransferListWidget]"/> QBT_TR(Limit upload rate...)QBT_TR[CONTEXT=TransferListWidget]</a></li>
+        <li><a href="#ShareRatio"><img src="theme/ratio" alt="QBT_TR(Limit share ratio...)QBT_TR[CONTEXT=TransferListWidget]"/> QBT_TR(Limit share ratio...)QBT_TR[CONTEXT=TransferListWidget]</a></li>
```

Comment:
for the img src, there is already one at icons/skin/ratio.png, the webUI cannot access there?

Reply:
That version is 256x256 and 5x larger than this icon.

Reply:
It's "usually" better to provide large image and scale it down than scale up small image, especially when high-dpi monitor is getting common.

Reply:
I meant 5x larger in terms of file size. In the WebUI this image has a static, specified height of 16px x 16px.

Reply:
>I meant 5x larger in terms of file size.

Surely our bottleneck isn't here... and almost all icons in the icon folder are 256x256, obviously you can see the motivation.
If you really insist adding the icon, move it to www/private/image folder.

>In the WebUI this image has a static, specified height of 16px x 16px.

Would be better if we used `em` as unit.

Reply:
> Would be better if we used em as unit.

This would be a great goal for a separate PR. I don't feel comfortable switching units for this small change.